pkg/kritis/util/util.go

```diff
@@ -62,12 +63,43 @@ func GetResource(image string) *grafeas.Resource {
 	return &grafeas.Resource{Uri: GetResourceURL(image)}
 }
 
-func GetPgpAttestationFromOccurrence(occ *grafeas.Occurrence) metadata.PGPAttestation {
-	pgp := occ.GetAttestation().GetAttestation().GetPgpSignedAttestation()
-	return metadata.PGPAttestation{
-		Signature: pgp.GetSignature(),
-		KeyID:     pgp.GetPgpKeyId(),
-		OccID:     occ.GetName(),
+func GetRawAttestationFromOccurrence(occ *grafeas.Occurrence) metadata.RawAttestation {
```

Comment:
Could you add some unit tests for this, as it grew somewhat complicated?

Reply:
Done.